src/modules/roc_audio/mixer.cpp

```diff
@@ -28,21 +28,31 @@ sample_t clamp(const sample_t x) {
 
 } // namespace
 
-Mixer::Mixer(core::BufferPool<sample_t>& pool, size_t frame_size)
-    : valid_(false) {
-    roc_log(LogDebug, "mixer: initializing: frame_size=%lu", (unsigned long)frame_size);
+Mixer::Mixer(core::BufferPool<sample_t>& pool, 
+             core::nanoseconds_t frame_length, 
+             size_t sample_rate, 
+             packet::channel_mask_t channel_mask)
+    : valid_(false)
+    , frame_size_((unsigned long)packet::timestamp_from_ns(frame_length, sample_rate) 
+                  * packet::num_channels(channel_mask)) {
```

Comment:
Shouldn't it be `size_t` instead of `unsigned long`? Here and in other places.

Reply:
done.